src/shadowsocks_config.ts

```diff
@@ -320,7 +316,7 @@ export const SIP002_URI = {
 
   stringify: (config: Config) => {
     const {host, port, method, password, tag, extra} = config;
-    const userInfo = Base64.encode(`${method.data}:${password.data}`);
+    const userInfo = Base64.encodeURI(`${method.data}:${password.data}`);
```

Comment:
It seems that Base64.decode() in the parse code handles both base64 and base64url, so that's good. But let's add a comment there saying it expects base64url as input.